public/app/plugins/datasource/loki/components/LokiQueryFieldForm.tsx

```diff
@@ -152,7 +152,7 @@ export class LokiQueryFieldForm extends React.PureComponent<LokiQueryFieldFormPr
 
     return (
       <>
-        <div className="gf-form-inline gf-form-inline--nowrap flex-grow-1">
+        <div className="gf-form-inline gf-form-inline--xs-view-flex-column flex-grow-1">
           <div className="gf-form flex-shrink-0">
```

Comment:
Remove nowrap. Instead of that, add custom class which changes flex-direction to column when we hit 'xs' breaking point.